Containerfile

```diff
@@ -1,4 +1,19 @@
 FROM registry.fedoraproject.org/fedora-minimal:37
+LABEL name="Cachi2" \
+      vendor="RHTAP Build Team" \
+      maintainer="container-build-guild@redhat.com" \
+      release="1" \
```

Comment:
More details in https://github.com/redhat-appstudio/hacbs-test/blob/main/policies/image/required-labels.rego (btw, version looks mandatory)

But it's a pretty funny check. It is based entirely around the labels required by OSBS, which auto-injects all of them except `com.redhat.component`, `name` and `version`

Reply:
Maybe we're not supposed to be adding them manually? Maybe the check is there for a future where the build pipeline injects them automatically?

What happens if we just ignore that check for now?

Reply:
Weird, the task was succesful in the CI that was triggered by this PR, even without the version :thinking:

Reply:
I checked with the EC folks (aka Lui), and apparently this is only required for images that are shipped to 
customers. I will try to get more information if they plan to make a task that will automatically inject these values.

The only reason I started this is because I saw the failing task, and it seemed like a quick fix :sweat_smile: